pkgs/stdenv/linux/default.nix

```diff
@@ -95,7 +107,7 @@ let
 
           # stdenv.glibc is used by GCC build to figure out the system-level
           # /usr/include directory.
-          inherit (prevStage) glibc;
+          # inherit (prevStage) glibc;
```

Comment:
Yay! `stdenv.glibc` is an awful attribute.

Reply:
Yes! Very glad you agree.  So far doesn't seem to be used for anything worthwhile.